tdrs-frontend/src/components/UploadReport/UploadReport.jsx

```diff
@@ -10,7 +10,7 @@ import { submit } from '../../actions/reports'
 import { useEventLogger } from '../../utils/eventLogger'
 import { fileUploadSections } from '../../reducers/reports'
 
-function UploadReport({ handleCancel, header, stt, submitEnabled }) {
+function UploadReport({ handleCancel, header, stt, ssp, submitEnabled  }) {
```

Comment:
prettier is really angry about the extra white space at the end of this line

Reply:
It also looks like submitEnabled isn't needed?